rails/app/views/layouts/application.html.haml

```diff
@@ -77,8 +77,6 @@
             = render :partial => 'shared/general_accordion_nav', :locals => {:top_node => @investigation, :selects => [@investigation,@activity,@section,@page]}
           - elsif @activity
             = render :partial => 'shared/general_accordion_nav', :locals => {:top_node => @activity, :selects => [@activity,@section,@page]}
-          - elsif @portal_school
-            = render :partial => 'shared/general_accordion_nav', :locals => {:top_node => @portal_school, :selects => [@school,@teacher,@portal_clazz,@portal_student]}
```

Comment:
There was a separate story about cleaning up this left hand nav. I just put a comment in that story that this PR took care of the school part:
https://www.pivotaltracker.com/story/show/174366238